pyproject.toml

```diff
@@ -12,6 +12,7 @@ packages = [
 python = "^3.9"
 griptape = "^1.0.0"
 python-dotenv = "^1.0.1"
+pyright = "^1.1.390"
```

Comment:
We already have pyright listed in dependencies. Please remove and then run `poetry lock --no-update`